src/components/common/InputAmount.vue

```diff
@@ -155,7 +160,26 @@ export default defineComponent({
       }
     };
 
-    return { arrUnitNames, update, isMaxAmount, isInitInput, initInput, isH160, setMaxAmount };
+    const isNominationTransferMax = computed(() => {
```

Comment:
I believe common component shouldn't know about nomination transfer or any other UI feature. Max amount should be provided as prop. In case we add another UI feature we will need to change InputAmount component again and risk to break some other feature

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/385/commits/e427de695ad9bed7cf426f78446fda72b1b85e37) commit.